docs/getting-started.md

```diff
@@ -121,6 +121,45 @@ npm run test:js:watch
 
 See `npm run test:js:help` to get a list of additional options that can be passed to the test runner.
 
+#### End-to-End Tests
+
+This project leverages the local Docker-based environment to facilitate end-to-end (e2e) testing using Puppeteer.
+
+To run the full test suite, you can use the following command:
+
+```bash
+npm run test:e2e
+```
+
+You can also watch for any file changes and only run tests that failed or have been modified:
+
+```bash
+npm run test:e2e:watch
+```
+
+Not using the built-in local environment? You can also pass any other URL to run the tests against. Example:
+
+```bash
+npm run test:e2e -- --wordpress-base-url=https://my-amp-dev-site.local
+```
+
+For debugging purposes, you can also run the E2E tests in non-headless mode:
+
+```bash
+npm run test:e2e:interactive
+```
+
+Note that this will also slow down all interactions during tests by 80ms. You can control these values individually too:
```

Comment:
Is 80ms the default or defined somewhere? Couldn't see where it's defined, checking just in case that it's accurate.

Reply:
It is defined here: https://github.com/WordPress/gutenberg/blob/ef73ed07bd59bdcbbe77cc1c537299d8e39167b1/packages/scripts/scripts/test-e2e.js#L50-L53